build.gradle.kts

```diff
@@ -35,6 +35,7 @@ dependencies {
     }
   }
 
+  implementation("org.flywaydb:flyway-core:7.10.0")
```

Comment:
Out of interest what does this change from `runtimeOnly` to `implementation` do? (Is it worth having a separate commit for it to explain the reasoning?)

Reply:
No a separate commit is not necessary for this.  I cannot write the code migration without it.  It was only available at runtime prior to this

This is standard stuff and probably more down to experience with Java/Kotlin/Gradle.

Reply:
I imagined it might be nice to have the commit split up into something like this:

```
Make flyway-core an implementation dependency

This is required to write database migrations in code. Previously it was
only available at runtime.
```

```
Add migration to assist with development

...[PR description here]...
```

Personally, I tend to prefer having lots of smaller commits as is it makes it easier to look back at the changes without the context of it being worked on right now. I think it's especially helpful for other developers who want to understand what's changed in the repo. I find it makes using tools like [`git bisect`](https://git-scm.com/docs/git-bisect) more successful as well.

I quite like [this article on commit messages](https://chris.beams.io/posts/git-commit/) which explains it much better than I could!